schema/alpha/docs/containers/workspace.md

```diff
@@ -5,7 +5,281 @@
 http://github.com/kubeflow/metadata/schema/alpha/containers/workspace.json
 ```
 
+schema for workspace, which is used to group artifacts and exectuions for solving a machine learning problem
 
 | Abstract | Extensible | Status | Identifiable | Custom Properties | Additional Properties | Defined In |
 |----------|------------|--------|--------------|-------------------|-----------------------|------------|
 | Can be instantiated | No | Experimental | No | Forbidden | Permitted | [containers/workspace.json](workspace.json) |
+## Schema Hierarchy
+
+*  `http://github.com/kubeflow/metadata/schema/alpha/containers/workspace.json`
+  * [entity](../entity.md) `http://github.com/kubeflow/metadata/schema/alpha/entity.json`
+
+
+##  Example
+```json
+{
+  "id": "4c2e8652-7286-11e9-a923-1681be663d3e",
+  "kind": "workspace",
+  "category": "container",
+  "namespace": "kubeflow.org",
+  "apiversion": "alpha",
+  "name": "MNIST",
+  "description": "workspace for MNIST problem",
+  "owner": "someone@kubeflow.org",
+  "create_time": "2018-12-13T20:20:39+00:00",
+  "annotations": {
+    "mylabel": "l1"
+  }
+}
+```
+
+#  Properties
+
+| Property | Type | Required | Nullable | Defined by |
+|----------|------|----------|----------|------------|
+| [annotations](#annotations) | `object` | Optional  | No | [entity](../entity.md#annotations) |
+| [apiversion](#apiversion) | `string` | **Required**  | No | [entity](../entity.md#apiversion) |
+| [category](#category) | `enum` | **Required**  | No | [entity](../entity.md#category) |
+| [create_time](#create_time) | `string` | Optional  | No | [entity](../entity.md#create_time) |
```

Comment:
Perhaps consider renaming to either "created_at" or "creation_time"?

Reply:
All these names make sense, but we have choose one convention. I'm following the Google could API design guide line [here].(https://cloud.google.com/apis/design/naming_convention#time_and_duration)

Reply:
Okay sounds good as long as we follow a particular convention.